test_pull_requests

```diff
@@ -458,6 +460,22 @@ module Hub
       comment += extended_tests if extended_tests
       recreate_comment(pull_id, bot_comment['id'], repo, comment)
 
+      bot_sha_comment = get_comment_with_prefix(pull_id, repo, BASE_SHA_PREFIX)
```

Comment:
I don't think you want a separate comment for this.  Would want to add the detail to the existing bot comment.